docs/user-guide/deploying-application/triggering-cd.md

```diff
@@ -10,9 +8,59 @@ Select an image to deploy and then click on `Deploy` to trigger the CD pipeline.
 
 ![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deploy.jpg)
 
-The current deployed images are tagged as `Deployed on <Environment name>`.
+The currently deployed images are tagged as `Active on <Environment name>`.
+
+## Manual Approval for Deployment
+
+When manual approval is enabled for the deployment pipeline, you are required to request image approval before each deployment. Alternatively, you can only deploy images that have already been approved once.
+
+When no approved image is available or the deployed image is already deployed, you will not see any image available to be deployed when you click on the `Select Image` button.
```

Comment:
When no approved image is available or if the image is already deployed, you will not see any image available for deployment upon clicking the `Select Image` button.